### PR TITLE
ConorOrr-signoff-navigation

### PR DESCRIPTION
**Pull request by arc2448**

### CODE REVIEW

File: Documentation/Signoffs/NavSystem/Detail Design.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Function of the Subsystem
+
+The Navigation System serves as the central control hub, responsible for directing the robot as it traverses its predefined course.
+The navigation system will use an array of infrared avoidance sensors to detect the yellow line. These sensors output low on black and dark colors, and high on white and light colors. The array will output digital bits
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/NavSystem/Detail Design.md]: Avoidance sensors? I'm not familiar with that term.

File: Documentation/Signoffs/NavSystem/Detail Design.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+## Function of the Subsystem
+
+The Navigation System serves as the central control hub, responsible for directing the robot as it traverses its predefined course.
+The navigation system will use an array of infrared avoidance sensors to detect the yellow line. These sensors output low on black and dark colors, and high on white and light colors. The array will output digital bits 
+
+## Constraints
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/NavSystem/Detail Design.md]: Put the constraints into a table and number them. Also, this seems like a short list. This is a sensing system so immediately we need constraints for sampling rate and the input (sensed colors).

**Review by JesseTNRoberts — COMMENTED** [review]
These aren't sufficiently detailed/accurate design schematics. Review past projects and the schematics there.

**Review by JesseTNRoberts — COMMENTED** [review]
As previously mentioned, this schematic is not buildable. It lacks significant detail regarding the necessary connections.

**Comment by arc2448** [comment]
I am not done with the schematic, I have been out of town.

**Review by JesseTNRoberts — COMMENTED** [review]
Nothing has been changed in the file. Review past schematics to see the level of detail required for electrical schematics. This is not sufficient to be built by a person solely from the schematic.

**Review by JesseTNRoberts — COMMENTED** [review]
Same feedback

**Comment by arc2448** [comment]
I am unsure how to make the electrical diagrams more specific as I do not have the schematic for the main controller. Do the changes I just committed look better?

**Review by JesseTNRoberts — COMMENTED** [review]
Have you considered how the placement of the sensor on the bot will affect the control system? 

How do we know that 20Hz is sufficient?

**Comment by arc2448** [comment]
I actually had that thought this morning and forgot to add it, thank you.

Added explination.

**Review by JesseTNRoberts — COMMENTED** [review]
How will it be mounted?

**Review by JesseTNRoberts — COMMENTED** [review]
Why are we not purchasing the one from pololu or amazon?

**Review by JesseTNRoberts — COMMENTED** [review]
Mounting is still not addressed. Need a buildable plan for mounting. Not a block diagram.

**Comment by arc2448** [comment]
To address the purchasing comment: The sensors from amazon or pololu are cheaper as the sensor array, but do not have an I2C output, and reliably converting the analog from 2 sensors to I2C would end up being the same cost or more expensive.

Will update model with mounting information

**Review by JesseTNRoberts — COMMENTED** [review]
We have talked about the changes to be made.

**Review by JesseTNRoberts — COMMENTED** [review]
I'm concerned that if you mount the sensor at the front of the bot, it will hit the ground on the inclines.

**Review by JesseTNRoberts — APPROVED** [review]
This is good and well reasoned.  

One thing that is a bit incorrect, but not a problem for the signoff is the idea that the sensor only needs 20hz since the motors update at this rate. 

This would be the case if you used only a "P" controlled. However, intermediate values (that are received between updates of the motor speed) will have a strong effect on the "I" and "D" values in the PID.

**Pull request merged**